packages/build/tests/helpers/temp.js

```diff
@@ -1,29 +1,23 @@
-'use strict'
+import { realpath } from 'fs'
+import { tmpdir } from 'os'
+import { normalize } from 'path'
+import { promisify } from 'util'
 
-require('log-process-errors/build/register/ava')
```

Comment:
This statement was removed since it is already called in https://github.com/netlify/build/pull/3967/files#diff-dba6d86e810144b67c8e92158abae43f3360e00524f2c5b4f87f4f47dcdb8baaL3